src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -0,0 +1,198 @@
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 58 to help you plan. #}
+
+# Deprecations and Removals in Chrome 58 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 58,
+which is in beta as of March 16. This list is subject to change at any time.
+
+
+## Mouse on Android stops firing TouchEvents
+
+Until Chrome 57, Android low-level mouse events in Chrome primarily followed an
+event path designed for touch interactions. For example, mouse drag motion while
+a mouse button is pressed generates `MotionEvents` delivered through
+`View.onTouchEvent`.
+
+However, since touch events cannot support hover, hovering mousemoves followed a
+separate path. The design had many side-effects including mouse interactions
+firing `TouchEvents`, all mouse buttons appearing as *left* mouse buttons, and
+`MouseEvents` being suppressed by `TouchEvents`.
+
+Starting with Chrome 58, a mouse on Android M or later will:
+
+* No longer fire `TouchEvents`.
+* Fire a consistent sequence of `MouseEvents` with appropriate buttons and
+  other properties. 
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/cNaFvMaYtNA/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5642080642662400) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=468806)
+
+
+## Remove case-insensitive matching for usemap attribute
+
+The `usemap` attribute was formerly defined as caseless. Unfortunately
+implementing this was complicated enough that no browsers implemented it
+correctly. Research suggested that such a complicated algorithm is unnecessary,
+and even ASCII case-insensitive matching is unnecessary.
+
+Consequently, the specification was updated so that case-sensitive matching is
+applied. The old behavior was deprecated in Chrome 57, and is now removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/8pHdFzN0YQc/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5760965337415680) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=659464)
+
+
+## Remove EME from unsecure contexts
+
+Some usages of
+[Encrypted Media Extenions (EME)](https://developer.mozilla.org/en-US/docs/Web/API/Encrypted_Media_Extensions_API)
+expose digital rights management implementations that are not open source,
+involve access to persistent unique identifiers, and/or run unsandboxed or with
+privileged access. Security risks are increased for sites exposed via insecure
+HTTP because they can be attacked by anyone on the channel. Additionally,
+permissions for an insecure HTTP site can be explited when explicit permissions
```

Comment:
Can you just rewrite this sentence for me? Your suggestion changes the grammar, and I do 't know enough about this subject rewrite this.